src/main/resources/interface-specification.yml

```
@@ -425,9 +604,16 @@ components:
         name:
           type: string
           pattern: /^[a-zA-Z]/s+$/
+        version:
+          type: string
+          pattern: ^[0-9]{1,4}$
+        state:
+          $ref: '#/components/schemas/EServiceState'
```

```suggestion
          $ref: '#/components/schemas/EServiceDescriptorState'
```
would be more accurate